app/views/smart_proxies/logs/_modal.html.erb

```diff
@@ -0,0 +1,40 @@
+<div class="modal fade" id="backtrace-modal" tabindex="-1" role="dialog" aria-labelledby="bt-modal-label">
```

Comment:
you can use the modal layout here instead of repeating all the modal boilerplate, take a look at https://github.com/theforeman/foreman/blob/develop/app/views/common/_modal.html.erb and https://github.com/theforeman/foreman/blob/develop/app/views/autosign/_form.html.erb#L2 for an example